src/components/OperatorList.tsx

```diff
@@ -78,13 +78,16 @@ const OperatorList: React.VFC<Props> = React.memo((props) => {
                   op.name
                 )}
               </span>
-              <span className="rarity" title={`Rarity: ${op.rarity + 1} stars`}>
-                <span className="rarity-number">{op.rarity + 1}</span>{" "}
+              <span className="rarity" aria-hidden="true">
+                <span className="rarity-number">{op.rarity + 1}</span>
                 <StarIcon className="rarity-star" />
               </span>
               <span key="opClass" className="operator-class">
                 {operatorClass}
               </span>
+              <span className="visually-hidden">{`Rarity: ${
+                op.rarity + 1
+              } stars`}</span>
```

Comment:
I wonder if this would work?
```html
<span className="rarity">
  <span className="visually-hidden">Rarity:</span>
  <span className="rarity-number">{op.rarity + 1}</span>{" "}
  <StarIcon className="rarity-star" aria-label="stars" />
</span>
```
(Where rather than marking all of `span.rarity` as `aria-hidden`, we instead insert some SR-only text to contextualize it)

Reply:
Yup that works better, updated.